pkg/proc/internal/ebpf/bpf/trace.bpf.c

```diff
@@ -244,4 +213,4 @@ int uprobe__dlv_trace(struct pt_regs *ctx) {
     return 0;
 }
 
-char _license[] SEC("license") = "GPL";
\ No newline at end of file
+char _license[] SEC("license") = "GPL";
```

Comment:
Unrelated to this PR but I didn't notice it before. Should this really say GPL?

Reply:
`bpf_probe_read_user` is used in this BPF program, and this helper is GPL only.

https://github.com/torvalds/linux/blob/8bb7eca972ad531c9b149c0a51ab43a417385813/kernel/trace/bpf_trace.c#L169-L176

Please also refer to https://github.com/torvalds/linux/blob/master/Documentation/bpf/bpf_licensing.rst.

Reply:
That says that to call GPL only functions the probe needs to be GPL compatible, which many licenses are.

Reply:
Right, but kernel only accept certain combinations.

https://github.com/torvalds/linux/blob/v5.15/include/linux/license.h#L5-L13

Reply:
We should rewrite it to be `Dual MIT/GPL`.